lavanyagarg112/meetwise
# Profile management

## Pull Request Template - Complete Profile management

**Description:**

 Add ability to delete users, meetings, organisations, update names,passwords, remove users from teams, organisations, meetings

**Related Issue:**

Resolves #88 from backend

**Changes Made:**

 Provide a detailed list of all changes made in this pull request. Be specific and use bullet points for readability.
 *  Refactor code 
 * Optimise code
 * Add aforementioned features

**How to Test:**

 Clearly explain how reviewers can test the new feature. Include specific steps and expected results.
 * Run old functions
 * Run new functions

**Checklist:**

 - [ ] Code compiles without errors and warnings.
 - [ ] Code follows project coding style and conventions.
 - [ ] Feature is fully functional and meets acceptance criteria.
